supportedBackends/htcondor/src/test/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActorSpec.scala

```diff
@@ -423,4 +453,11 @@ class HtCondorJobExecutionActorSpec extends TestKitSuite("HtCondorJobExecutionAc
     override def storeExecutionResult(cachedExecutionResult: CachedExecutionResult): Unit = ()
   }
 
+  class KVServiceActor extends Actor {
```

Comment:
Yep.. they have a fairly standard `.expectMsg(..)` and `.reply(..)` methods to do exactly what you are doing here.